### PR TITLE
Detach bpf code on SIGTERM/SIGINT signal

### PR DESCRIPTION
**Pull request by sbernard31**
It aims to fix https://github.com/AirVantage/sbulb/pull/10#discussion_r304480681.

### CODE REVIEW

File: ulb.py [hunk 1]
```diff
@@ -197,40 +198,56 @@ def print_event(cpu, data, size):
...
 
+# Handle Signal
+class Stopper:
+  _stop = False
+  def __init__(self):
+    signal.signal(signal.SIGTERM, self.stop)
+    signal.signal(signal.SIGINT, self.stop) # keyboard interruption
+
+  def stop(self,signum, frame):
+    print ("\nStopping by signal {}({})...".format(signal.Signals(signum).name, signum));
```
Comment (oliwer) [inline ulb.py]: s/Stopping by/Received/

File: ulb.py [hunk 2]
```diff
@@ -197,40 +198,56 @@ def print_event(cpu, data, size):
...
+        #(task, pid, cpu, flags, ts, msg) = b.trace_fields(nonblocking = True)
+        #while msg:
+        #    print("%s \n" % (msg))
+        #    (task, pid, cpu, flags, ts, msg) = b.trace_fields(nonblocking = True)
 finally:
     # Detach bpf progam
+    print ("Detaching bpf code ...")
     b.remove_xdp(ifnet)
+    print ("... code detached.")
+    print ("... sbubl stopped.")
```
Comment (oliwer) [inline ulb.py]: You misspelled sbulb :scream:

File: ulb.py (hunk 1 above)
Comment (sbernard31) [reply ulb.py]: I want a message which show we are in a process about stopping the application.
So I prefer Stopping

File: ulb.py (hunk 2 above)
Comment (sbernard31) [reply ulb.py]: I will fix this.

**Pull request merged**